nextcloudappstore/core/templates/user/app-ownership-transfer.html

```diff
@@ -0,0 +1,42 @@
+{% extends "user/base.html" %}
+{% load i18n %}
+{% load staticfiles humanize %}
+
+{% block head-title %}{% trans 'App ownership transfers' %} - {% endblock %}
+
+{% block account-content %}
+    <h1>{% trans "Pending app ownership transfers" %}</h1>
+    <section>
+        {% for transfer in object_list %}
+            <h2><a href="{% url 'app-detail' transfer.app.id %}">{{ transfer.app.name }}</a></h2>
+            {% comment %}Translators: 'datetime' is rendered with the 'naturaltime' filter: https://docs.djangoproject.com/en/1.10/ref/contrib/humanize/{% endcomment %}
+            <p>{% blocktrans with datetime=transfer.proposed|naturaltime %}Transfer proposed {{ datetime }}{% endblocktrans %}</p>
```

Comment:
In that case I'd drop the naturaltime. As I see it, the point of it is to express things in natural language (without ":"), and therefore I chose blocktrans so that translators could have greater freedom in forming natural sentences in each of their languages.